packages/atomic/src/components/atomic-modal/atomic-modal.tsx

```diff
@@ -90,55 +105,55 @@ export class AtomicModal implements InitializableComponent {
     }
 
     return (
-      <div
-        part="backdrop"
-        class={`fixed left-0 top-0 right-0 bottom-0 z-10 ${
-          this.isOpen ? 'opened' : ''
-        }`}
-        onClick={(e) => e.target === e.currentTarget && this.close()}
-      >
-        <atomic-focus-trap
-          active={this.isOpen}
-          role="dialog"
-          aria-modal={this.isOpen.toString()}
-          aria-labelledby={this.headerId}
+      <Host class={this.getClasses().join(' ')}>
```

Comment:
I recommend that you ensure that whitespace changes are hidden to review this section, otherwise you will see a huge diff.
![image](https://user-images.githubusercontent.com/54454747/166294123-0161bf2c-a0e6-4618-a825-58da6f463456.png)

I moved the `opened` class to the `Host`.